apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -240,13 +240,30 @@ type CWSFeatureConfig struct {
 	// +optional
 	SyscallMonitorEnabled *bool `json:"syscallMonitorEnabled,omitempty"`
 
+	Network          *CWSNetworkConfig          `json:"network,omitempty"`
+	SecurityProfiles *CWSSecurityProfilesConfig `json:"securityProfiles,omitempty"`
+
 	// CustomPolicies contains security policies.
 	// The content of the ConfigMap will be merged with the policies bundled with the agent.
 	// Any policies with the same name as those existing in the agent will take precedence.
 	// +optional
 	CustomPolicies *CustomConfig `json:"customPolicies,omitempty"`
 }
 
+type CWSNetworkConfig struct {
+	// Enabled enables Cloud Workload Security Network detections.
+	// Default: false
```

Comment:
Should this default be `true` based on `defaultCWSNetworkEnabled` above?

Reply:
Yes it should, fixed in last commit